javascript/javascript1/week2/homework-js1-w2.js

```diff
@@ -5,27 +5,32 @@
 // Flight booking fullname function
 //*****************************************/
 
-function getFullName(firstName, surName, useFormalName = false, gender) {
+function getFullName(firstName, surName, useFormalName = false, isMale) {
+  // Data
+  const preFixMale = "Lord";
+  const preFixFemale = "Lady";
+
+  // Logic
   if (!firstName.trim() || !surName.trim()) {
     return "Please provide both first name and surname.";
   }
 
   const fullName = `${firstName} ${surName}`;
-
   if (!useFormalName) {
     return fullName;
   } else {
-    if (gender === "man") {
-      return `Lord ${fullName}`;
-    } else if (gender === "woman") {
-      return `Lady ${fullName}`;
+    if (isMale) {
+      return `${preFixMale} ${fullName}`;
+    } else if (!isMale) {
```

Comment:
Could be simplified a bit
```suggestion
    } else {
```

Reply:
So fast review! Thank you! 😊